src/Features/Core/Portable/SolutionCrawler/IDocumentDifferenceService.cs

```diff
@@ -8,13 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.SolutionCrawler;
 
-internal class DocumentDifferenceResult(InvocationReasons changeType, SyntaxNode? changedMember = null)
-{
-    public InvocationReasons ChangeType { get; } = changeType;
-    public SyntaxNode? ChangedMember { get; } = changedMember;
```

Comment:
the old system would return both the Changed-Member *and* the type of change it was.  The latter was used by solution crawler to adjust crawling.  Since that's no longer a thing, this is changed to just return the changed-member.